src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetUnallocatedCaseByTeamTests.kt

```diff
@@ -86,4 +87,47 @@ class GetUnallocatedCaseByTeamTests : IntegrationTestBase() {
       .expectStatus()
       .isUnauthorized
   }
+  @Test
+  fun `not found`() {
```

Comment:
I don't see the point of this test in this file. In fact isn't this just testing Spring Boot?

I see it's been copied but still...